lib/client.js

```diff
@@ -349,12 +349,25 @@ Client.prototype.loadMetadataForTopics = function (topics, cb) {
 };
 
 Client.prototype.createTopics = function (topics, isAsync, cb) {
+  function isValid (str) {
+    return !/[~`!#$%\^&*+=\-\[\]\\';,/{}|\\":<>\?]/g.test(str);
+  }
   topics = typeof topics === 'string' ? [topics] : topics;
 
   if (typeof isAsync === 'function' && typeof cb === 'undefined') {
     cb = isAsync;
     isAsync = true;
   }
+  var validTopics = true;
+  Object.keys(topics).map(function (elem) {
+    if (!isValid(topics[elem])) {
+      validTopics = false;
+    }
+  });
+
+  if (!validTopics) {
+    return cb(null, 'Some topics contains special characters, please remove them');
```

Comment:
Lets continue with the error first style and call the callback with a new `Error` as the first argument.